### ISSUE TITLE
more config

### ISSUE THREAD

**Issue by gilcot** [alextremblay/phpipam_scraper#6]
Hello

Why not have in configuration file the paramaters `username` and `password` so they are not asked (like url) when not provided in command line options? I prefer, in cron scripts, refer to a file with a proper permissions than supplying the password in the script file. 

Thanks.

**Comment by alextremblay** [comment]
You're absolutely right. I'm going to start working on this. I'll reply here when done.
